File: servidor/src/main/java/com/uniquindio/retos/dto/LogDTO.java
package com.uniquindio.retos.dto;

import lombok.*;

import java.time.LocalDateTime;

@AllArgsConstructor
@Setter
@Getter
@NoArgsConstructor
public class LogDTO {

    private String application;

    private String logType;

    private String module;

    private LocalDateTime dateTime;

    private String summary;

    private String description;

}

File: logs/src/main/kotlin/com/uniquindio/reto_1/dto/logDTO.kt
package com.uniquindio.reto_1.dto

import java.time.LocalDateTime

data class LogDTO(

    var application: String = "",
    var logType: String = "",
    var module: String = "",
    var dateTime: LocalDateTime = LocalDateTime.now(),
    var summary: String = "",
    var description: String = ""
){}
